command/init.go

```diff
@@ -76,7 +76,7 @@ for more info.`)
 	}
 
 	opts := plugingetter.ListInstallationsOptions{
-		FromFolders: c.Meta.CoreConfig.Components.PluginConfig.KnownPluginFolders,
+		PluginDirectory: c.Meta.CoreConfig.Components.PluginConfig.PluginDirectory,
```

Comment:
I don't have a better name than `FromFolders` at the moment. But I do find the repetitive use of PluginDirectory a bit confusing. Maybe `Src`,`SourceDirectory`, or just `Directory` is an option.